### ISSUE TITLE
Crash (reboot) when I try to run w3m

### ISSUE THREAD

**Issue by isrgish** [termux/termux-packages#2244]
After installing w3m I tried to first run it with a file on my device and it crashed my device (the device rebooted). I then tried to run it with help and again without any args. Each time my device rebooted.

I'm on a RCA Voyager device running Android 6

**Comment by xeffyr** [comment]
> After installing w3m I tried to first run it with a file on my device and it crashed my device (the device rebooted). I then tried to run it with help and again without any args. Each time my device rebooted.

Possible that this is Android's bug, non-root programs shouldn't cause critical errors that will make device reboot.

**Comment by isrgish** [comment]
Is there anything I can do here to correct the problem? I understand that others are using it without problem.

By the way my device is rooted.

**Comment by sdrausty** [comment]
```
[16:53 ~ ]$ w3m http://termux.com | head
Termux

Termux is an Android terminal emulator and Linux environment app that works
directly with no rooting or setup required. A minimal base system is installed
automatically - additional packages are available using the APT package
manager.

Get it on Google Play Get it on F-Droid

Read the wiki to learn more
[18:34 ~ ]$ uname -mo
aarch64 Android
```

@isrgish have you run `pkg up`?  What is your Termux version in apps in Android?  

> the device rebooted

Were there other apps and process running in Termux at the time?  
 Run `termux-info` and share more for this jigsaw puzzle to grow :a:

**Comment by isrgish** [comment]
I ran apt update.
My termux version is 0.60

The first time I ran it I may have had other apps running, but after that I ran it right after it rebooted.

This is the results of termux-info
[u0_a117 …/~]$ termux-info
Updatable packages:
curl/stable 7.59.0 i686 [upgradable from: 7.58.0]
libcurl/stable 7.59.0 i686 [upgradable from: 7.58.0]
vim/stable 8.0.1600 i686 [upgradable from: 8.0.1550]
vim-runtime/stable 8.0.1600 all [upgradable from: 8.0.1550]
System information:
Linux localhost 3.14.0+ #234 SMP PREEMPT Fri Mar 31 14:44:45 CST 2017 x86_64 Android
Termux-packages arch:
i686
Android version:
6.0.1
Device manufacturer:
Alco
Device model:
RCT6873W42
Terminated
[u0_a117 …/~]$

I upgraded now again to upgrade what I see is upgrade-able.
